mtupper/homework.md

```diff
@@ -0,0 +1,292 @@
+### Homework #1
+
+(Due Monday 1/12)
+
+---
+
+**1)** (_Difficulty: mixed_)
+Predict the output of each of the following expressions, and explain the rules which determine it.
+If the output depends on the value of x, identify the conditions when the expression will output true.  Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.
+
+**a)** `"1" == 1`<br />
+> **true** - string 1 is "similar enough" to number 1<br />
+
+**b)** `"1" === 1`<br />
+> **false** - string 1 is, strictly speaking, not the same as number 1<br />
+
+**c)** `x == 'x'`<br />
+> **false** - the only way to make this true would be to declare a variable x with a value of the string 'x'<br />
+> `var x = "x"`<br />
+
+**d)** `x == (x+'')`<br />
+> **true** - since adding nothing (empty string `''`) to the variable x won't change its value<br />
```

Comment:
Careful!  String-ifying a few values (NaN, undef, true, false) does make them unequal (according to ==)